baseframe/forms/auto.py

```diff
@@ -26,25 +26,37 @@ def render_form(form, title, message='', formid='form', submit=__(u"Submit"), ca
     else:
         code = 200
     if request.is_xhr and ajax:
-        return make_response(render_template('baseframe/ajaxform.html', form=form, title=title,
+        template = 'baseframe/bootstrap3/ajaxform.html.jinja2'
+        if current_app.config['theme'] == "mui":
```

Comment:
`'mui'` in single quotes everywhere as it's an identifier, not a UI string.

Reply:
Removed.